### PR TITLE
Fix experimental layers: mesh layer Y-reversal issue

### PR DESCRIPTION
**Pull request by georgios-uber**

### CODE REVIEW

File: modules/experimental-layers/src/path-marker-layer/path-marker-layer.js [hunk 1]
```diff
@@ -49,11 +49,11 @@ export default class PathMarkerLayer extends CompositeLayer {
     if (coordinateSystem === COORDINATE_SYSTEM.METER_OFFSETS) {
       const [dx, dy] = viewport.metersToLngLatDelta(xyz);
       const [x, y] = coordinateOrigin;
-      return viewport.projectFlat([x - dx, dy + y]);
+      return viewport.projectFlat([x + dx, dy + y]);
```
Comment (ibgreen) [inline modules/experimental-layers/src/path-marker-layer/path-marker-layer.js]: Add a HACK/TODO comment?

Comment (georgios-uber) [reply modules/experimental-layers/src/path-marker-layer/path-marker-layer.js]: The previous code was the "hacky" one. This one is the proper one.

Comment (georgios-uber) [reply modules/experimental-layers/src/path-marker-layer/path-marker-layer.js]: Yes, when we fix the `project module`, this will remain the same.

**Comment by georgios-uber** [comment]
@ibgreen I'm afraid if I _fix_ the projection logic, it might break (or maybe fix?) other layers.

**Pull request merged**